### PR TITLE
fix (zkquiz): update to new sdk version

### LINKED ISSUE #930: fix (zkquiz): update to new sdk version
Add sender address

### PR DESCRIPTION
**Pull request by taturosati**
> [!NOTE]
> This PR has been included in the V0.6.0 release
> This PR is thus marked as DRAFT for a short time, just in case
> But it should not be merged

To test:

```
cd examples/zkquiz
make answer_quiz KEYSTORE_PATH=<keystore_path>
```

### CODE REVIEW

File: docs/3_guides/2_integrating_aligned_into_your_application.md [hunk 1]
```diff
@@ -147,13 +148,33 @@ async fn submit_proof_to_aligned(
...
-        .map_err(|e| anyhow::anyhow!("Failed to submit proof: {:?}", e))
+    match submit_and_wait_verification(
+        BATCHER_URL,
+        &rpc_url,
+        Chain::Holesky,
+        &verification_data,
+        wallet.clone(),
+        nonce,
+        BATCHER_PAYMENTS_ADDRESS
+    )
```
Comment (NicolasRampoldi) [inline docs/3_guides/2_integrating_aligned_into_your_application.md:159]: Repeated function call

**Review by NicolasRampoldi — COMMENTED** [review]
Small comment, other than that it works correctly and code seems good

**Review by JuArce — COMMENTED** [review]
Same as Nico. Lgtm

**Comment by MauroToscano** [comment]
This is a hotfix for the testnet, we can have one PR for main, but first merge it there

**Comment by uri-99** [comment]
Closed after staging / testnet branch are stable

**Pull request closed without merging**